care/emr/reports/context_builder/data_points/base.py

```
@@ -0,0 +1,130 @@
+            return [
+                self.__class__(is_preview=True)
+                for c in range(random.randint(1, limit))  # noqa S311
+            ]
+        return self._filter(**kwargs)
+
+
+class QuerysetContextBuilder(ContextBuilderBase):
+    __context_type__ = "queryset"
+
+    filterset_class = None
+    __filterset_backends__ = []
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)